### ISSUE TITLE
New release please :)

### ISSUE THREAD

**Issue by wspr** [phst/lualatex-math#21]
I've just pushed some changes to keep lualatex-math up to date with expl3. The current code isn't broken but calls errors if expl3 is loaded in debug mode (which I'm trying to use for all my testing).

Do you have the capacity to send a new version through to CTAN? If not, I can do it but I think I'll need an email from you authorising me as a maintainer first.

**Comment by phst** [comment]
I'm happy to push a release, but maybe you want to try first? I'm mostly curious what happens, since I don't see any form of authorization on CTAN.

**Comment by davidcarlisle** [comment]
ctan authorisation works via (unlisted) email addresses of the authorised
uploaders. They will not accept updates from a "new" person (even someone
they know) without authorisation from the listed maintainer.

**Comment by wspr** [comment]
Yup. I also contacted them separately and they confirmed I wasn’t listed as an approved uploader (but I’m happy to be if you like, doesn’t mean I need to be the official maintainer of the package)

**Comment by wspr** [comment]
Thanks, very much appreciated!
